### ISSUE TITLE
Method it() outputs the word "it" thus preventing clean localization

### ISSUE THREAD

**Issue by kilathaar** [mscharhag/oleaster#26]
If the method only echoed the descriptive string parameter the problem would be solved. Pretty sure Jasmine does it like this as well.

**Comment by peterklijn** [comment]
Not only for localization, also for flexibility in writing tests:

```java
describe("The Product class", () -> {
    it("should be awesome", () -> {
        Product p = new Product();
        expect(p.areYouAwesome()).toBeTrue();
    });
});
```
This now outputs as `The Product class, it should be awesome`. Which breaks the natural sentence imho. `The Product class should be awesome` wouldn't (arguably even without the comma)

**Comment by mscharhag** [comment]
Yeah I can see the issue. Automatically adding `it` in the description wasn't the best idea.

**Comment by holmjoel** [comment]
Any idea when 0.2.1 will be released?

The output of "it" actually made me change to https://github.com/greghaskins/spectrum instead. Small thing but it really threw me off when running the tests.

**Comment by bangarharshit** [comment]
@joellundell I have fixed the issue in my fork. @mscharhag Just wanted to see if you will be open to PR for these issues as I don't have to maintain a separate fork.

**Comment by markjohndoyle** [comment]
I suppose the intent was to write tests something like this:

```java
describe("When the Product class", () -> {
    describe("does x", () -> {
        it("should be awesome", () -> {});
    });
    describe("does y", () -> {
        it("should not be quite so awesome", () -> {});
        });
    });
```

**Comment by raner** [comment]
There is another reason why `it` is problematic: in some JVM-based languages, like Kotlin and Xtend, `it` is a reserved keyword for a single implicit parameter. Not sure about Kotlin, but at least in Xtend there seems to be a way to work around it, but it's bound to cause issues and extra work in those languages.
